package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "NecroBot2-GUI",
-  "version": "1.0.0",
+  "version": "1.0.0.118",
```

Comment:
Yes it needs to be increased on each update, in order to register in squirrel as an update